tests/Backend/Synapse/CreateTableTest.php

```diff
@@ -0,0 +1,90 @@
+<?php
+
+namespace Keboola\Test\Backend\Synapse;
+
+use Keboola\Csv\CsvFile;
+use Keboola\StorageApi\ClientException;
+use Keboola\Test\StorageApiTestCase;
+
+class CreateTableTest extends StorageApiTestCase
+{
+    public function setUp()
+    {
+        parent::setUp();
+        $this->_initEmptyTestBuckets();
```

Comment:
```suggestion
        $this->initEmptyTestBucketsForParallelTests();
```
Tu by to mohlo byt rovno pripravene na parelelny test co myslis ?